guides/common/modules/proc_installing-webhooks-shellhooks-plugin.adoc

```diff
@@ -0,0 +1,26 @@
+[id="installing-webhooks-shellhooks-plugin_{context}"]
+= Installing Webhooks and Shellhooks Plugin
+
+.Procedure
+
+
+* Install the webhooks plugin using the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# foreman-installer --enable-foreman-plugin-webhooks
+----
+
+* Optionally, install the shellhooks plugin on each {SmartProxy} that is used for shellhooks, using the following command:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# foreman-installer --enable-foreman-proxy-plugin-shellhooks
```

Comment:
You need to add an attribute for foreman-installer
https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L2